src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -387,20 +387,27 @@ protected int doWork() {
     protected String[] customCommandLineValidation() {
 
         try {
-            final boolean isBamOrSamFile = isBamOrSam(IOUtil.getPath(INPUT));
-            if (!isBamOrSamFile && IGNORE_READ_GROUPS) {
-                return new String[]{"The parameter IGNORE_READ_GROUPS can only be used with BAM/SAM inputs."};
+            final boolean isBamOrSamOrCramFile = isBamOrSamOrCram(IOUtil.getPath(INPUT));
+            if (!isBamOrSamOrCramFile && IGNORE_READ_GROUPS) {
```

Comment:
this is becomming silly, can we change the variable to something like `fileContainsReads`?